tests/acceptance-tests/example_apps/print_request_headers/main.go

```diff
@@ -0,0 +1,29 @@
+package main
```

Comment:
I think `example_apps` directory was just an example. Shall we put this app in `apps` dir instead like here: https://github.com/cloudfoundry/cf-acceptance-tests/tree/master/apps ?